frontend/src/app/forms/forms.page.html

```diff
@@ -41,15 +41,22 @@
     >
     </form-basic-info>
   </ng-container>
+  <ng-container [ngSwitch]="currentMenu.selector">
+    <app-submit-page
```

Comment:
Will be better if all the form step component selector follow a common structure like form-<comp_purpose>

Reply:
done